src/app/GitExtUtils/GitUI/Interops/COMBOBOXINFO.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Runtime.InteropServices;
+
+namespace System;
+
+internal static partial class NativeMethods
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public struct COMBOBOXINFO
```

Comment:
We use to avoid publishing too much. (There may exist older code using `public` yet.) Applies to all added structs, enums and constants. 
```suggestion
    internal struct COMBOBOXINFO
```